internal/testutil/errors.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright Avast Software. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package testutil
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// RequireErrorContains error helper testing function.
+// nolint:thelper,nolintlint //it isn't helper function it is require function.
```

Comment:
While I understand the reason for the `thelper` nolint, having line 15 say that this is a helper testing function and then immediately after on line 16 say that it isn't a helper function is pretty contradictory. I would suggest changing line 15 to just briefly describe what the function does to get around it.